### PR TITLE
puzzle_simple_5

### PR DESCRIPTION
**Pull request by P-Rama-Ganesh**
# Your checklist for this pull request

## About your game

What is your game about? 
Pushing boxes to the goal. (from [Sokoban Plus](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/sokoban_plus.js))

How to play your game?
Capture
ou keyboard to restart the level
 Press WASD to move, J to restart and K to toggle trails, Get A boxes (cyan) to A goals (green), Get B boxes (magenta) to B goals (red), Get normal boxes (gray) to either goal. (from [Sokoban plus](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/sokoban_plus.js))

## Code
Check off the items that are true.
- [x] The game was made using the [Sprig editor](https://editor.sprig.hackclub.com/).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [x] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [x] The name of the image matches the name of your file. Example: `sokoban_plus.js` -> `sokoban_plus.png`.

> Thanks for PR!

### CODE REVIEW

**Comment by LucasHT22** [comment]
Thanks for PR! How about adding a little more to your Dodging game? Is the name My_First_Game really the name? if you need help, call me :)

**Comment by P-Rama-Ganesh** [comment]
I actually kept my game  as puzzle_simple_5

**Comment by LucasHT22** [comment]
I can't solve the last 2 levels, could you show me how to solve it?

**Comment by SamDev-7** [comment]
You're uploading four games, are you sure you want to do that?
Also, `puzzle_simple_five.js` isn't in the games directory, it needs to be moved. 
The test is also failing, try to fix it and let us know if you need any help!
Happy hacking!

**Comment by P-Rama-Ganesh** [comment]
yes i need help

**Comment by leomcelroy** [comment]
@P-Rama-Ganesh were you able to get the help you needed! There are lots of people active in the #sprig channel on the [Hack Club Slack](https://hackclub.com/slack/) if you need more.

File: puzzle_simple_5.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
+/*
+@title: puzzle_simple_5
+@author: P. Rama Ganesh
+
```
Comment (SamDev-7) [inline puzzle_simple_5.js:4]:
```suggestion
*/

```
You forgot to close the comment!

**Review by SamDev-7 — CHANGES_REQUESTED** [review]
Just to confirm, you want to submit `puzzle_simple_5.js`. If so, can you remove the other games?
Please move `puzzle_simple_5.js` to the `games/` directory. This can be done by renaming the file to `games/puzzle_simple_5.js`.
Please also rename the image to `puzzle_simple_5.png`.

You're so close to getting a sprig!

**Comment by grymmy** [comment]
@P-Rama-Ganesh I am closing this PR.  **You are free to reopen this PR yourself** (or reach out to me directly!) and continue advocating for this code landing, but it seems that it hasn't been acted upon for 3 months.

**Pull request closed without merging**